scripts/exampleSetup.ts

```diff
@@ -119,97 +88,59 @@ async function main() {
   await sdl.approve(veSDL.address, MAX_UINT256)
 
   // Create max lock with 10M SDL
-  const create_lock_gas_estimate = await veSDL.estimateGas.create_lock(
-    BIG_NUMBER_1E18.mul(10_000_000),
-    (await getCurrentBlockTimestamp()) + 4 * YEAR,
-  )
```

Comment:
Instead of actually trying to estimate, we can just use arbitrary 3_000_000 as safe gas limit for testing purposes.